src/Init/Data/BitVec/Basic.lean

```diff
@@ -767,6 +767,15 @@ SMT-Lib name: `bvnego`.
 def negOverflow {w : Nat} (x : BitVec w) : Bool :=
   x.toInt == - 2 ^ (w - 1)
 
+/--
+Checks whether the signed division of `x` by `y` results in overflow.
+For BitVecs `x` and `y` with width `0 < w`, this happens iff `x = intMin` and `y = allOnes w`.
```

Comment:
```suggestion
For BitVecs `x` and `y` with nonzero width, this only happens if `x = intMin` and `y = allOnes w`.
```

Following the comment style for `negOverflow`.

Reply:
fixed